google/resource_bigtable_instance.go

```diff
@@ -44,11 +44,17 @@ func resourceBigtableInstance() *schema.Resource {
 			},
 
 			"num_nodes": {
-				Type:         schema.TypeInt,
+				Type:     schema.TypeInt,
+				Optional: true,
+				ForceNew: true,
```

Comment:
Can we leave the `num_nodes` default to 3 and just don't send the property when creating a `DEVELOPMENT` instance? This way, we don't create churn for our users...